docker/Dockerfile-prod

```diff
@@ -11,6 +11,8 @@ CMD ["/usr/local/bin/uwsgi"]
 
 RUN addgroup -g 10001 app && adduser -D -u 10001 -G app -h /app app
 
+ENV PYTHONUNBUFFERED=1
```

Comment:
Yes, in prod it's even more important to not lose output because of a buffer.  You never know when there will be a message in a buffer somewhere that's critical to debugging a problem, and you'll spend days pulling your hair out trying to figure out why there is no error message before the crash. :smile: :cry: